.env.example

```
@@ -44,6 +44,8 @@ UPSTASH_REDIS_REST_TOKEN=your-upstash-token
 QSTASH_TOKEN=your-qstash-token
 QSTASH_CURRENT_SIGNING_KEY=your-qstash-current-signing-key
 QSTASH_NEXT_SIGNING_KEY=your-qstash-next-signing-key
+# Set to false to fail closed when Redis is unavailable (idempotency keys)
+IDEMPOTENCY_FAIL_OPEN=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The IDEMPOTENCY_FAIL_OPEN key should go before the QSTASH_CURRENT_SIGNING_KEY key

(UnorderedKey)